packaging/rpm/get-vendored-sources.sh

```
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -oeux pipefail
+
+NAME="flightctl"
+SPEC="$(rpmspec -P "./packaging/rpm/${NAME}.spec")"
+VERSION="$(grep '^Version:' <<< "${SPEC}" | awk '{print $2}')"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, here-strings are undefined.

(SC3011)